npm/qsharp/ux/circuit-vis/utils.ts

```diff
@@ -327,6 +327,20 @@ const getGateElems = (container: HTMLElement): SVGGraphicsElement[] => {
     : [];
 };
 
+// Non-ASCII chars are fraught with danger. Copy/paste these when possible.
+// Use the following regex in VS Code to find invalid unicode chars
+// [^\x20-\x7e\u{03b8}-\u{03c8}\u{2020}\u{2223}\u{27e8}\u{27e9}]
+
+const mathChars = {
+  theta: "θ", // \u{03b8}
+  pi: "π", // \u{03c0}
+  psi: "ψ", // \u{03c8}
+  dagger: "†", // \u{2020}
+  divides: "∣", // \u{2223}
```

Comment:
Actually, it *does* seem like we are using the "vertical line" | everywhere except here and in the comment you edited in gateFormatter.ts. It'd be confusing a probably lead to inconsistencies to keep this in here as it would indicate that this is the character we use for bra-ket notation when it isn't. Alternatively, we could change everywhere where we're using the vertical line, but that'd be a bigger change.

Reply:
I'm going to take this out. KaTeX uses that special bar char shown here, but when I used it in Jupyter Notebooks (which use a different font) it looked terrible in Safari, so I went back to just the regular ASCII bar char, which looks OK everywhere. I'll remove this.